### DISCUSSION TITLE
Add RJCP SerialPort implementation

### DISCUSSION THREAD

**Discussion by arkypita** [arkypita/LaserGRBL#2439] [General]
This release add the implementation of [RJCP SerialPort](https://github.com/jcurl/RJCP.DLL.SerialPortStream) an independent implementation of System.IO.Ports.SerialPort. 

User can now select this Com Wrapper from LaserGRBL configuration if other wrapper (UsbSerial and UsbSerial2) fails.

This wrapper also get rid off the CH340 driver issue: [https://lasergrbl.com/november-2024-windows-driver-update/](https://lasergrbl.com/november-2024-windows-driver-update/)

![image](https://github.com/user-attachments/assets/e445b48c-c635-4941-be29-92b38e4f235c)

<hr /><em>This discussion was created from the release <a href='https://github.com/arkypita/LaserGRBL/releases/tag/v7.13.0'>Add RJCP SerialPort implementation</a>.</em>

**Comment by ristraus** [comment]
I did a test with a file, which took 40 min to engrave. No problems with the new RJCP serialport connection. Just for test, loaded it as well under the original USB connection (CH340) and notice that the execution time was exactly the same.

Looking forward for the .resx file prior to final release to all users, so I can update the PT-BR translations.

**Reply by ristraus** [reply]
Did another job today, about 2h10m, no errors.

**Comment by Plomobe** [comment]
My laser engraver was unresponsive when I connected using V7.12 due to the Windows update CH430 driver issue.
Using the RJCP Serial connection protocol made the engraver responsive again!

Keep up the good work.

**Comment by jbirdlebough1** [comment]
Installed on W10, rebooted, device manager reports port active.  Lasergrbl sees com7, but would not connect after driver upgrade. Ch340 works still.
